execution-environment/README.md

```diff
@@ -10,13 +10,13 @@ Tools and configuration to run playbooks that interact with both OpenStack/ESI a
     uv pip compile ../pyproject.toml > requirements.txt
     ```
 
-1. Download the `openshift-clients` package from [the customer portal] and place it in
+2. Download the `openshift-clients` package from [the customer portal](https://access.redhat.com/downloads/content/290/ver=4.18/rhel---9/4.18.9/x86_64/packages) and place it in
     `openshift-clients.rpm`
 
-2. Run:
+3. Grab your Automation Hub token from [Red Hat console](https://console.redhat.com/ansible/automation-hub/token)
+
+4. Run:
 
     ```
-    ansible-builder build --tag cloudkit-aap-ee
+    ansible-builder build --tag cloudkit-aap-ee --build-arg ANSIBLE_GALAXY_SERVER_AUTOMATION_HUB_TOKEN=<your Automation Hub token>
```

Comment:
How does this whole workflow compare with just maintaining a Containerfile and building it like any other container image?

Reply:
the `ansible-builder` generates a container file and builds an image. I would say it's not too bad, as it defaults a lot of stuff.  It also takes into account the dependencies of the installed collections (dependencies can be python dependencies,  or anything as bindep format, like rpms), installing them is a bit more involved than `ansible-galaxy collection install ....`.

You can explorer the build artifacts by running `ansible-builder`.